controllers/controller.js

```diff
@@ -10,4 +10,10 @@ const listTopics = (request, response) => {
     })
 }
 
-module.exports = { listTopics, handle404Errors }
\ No newline at end of file
+const listArticles = (request, response) => {
+    findArticles().then((articles) => {
+        response.status(200).send(articles)
```

Comment:
we need to send responses back to the client on an appropriate key eg. {articles}